racket.html.markdown

```diff
@@ -8,6 +8,7 @@ contributors:
   - ["Gustavo Schmidt", "https://github.com/gustavoschmidt"]
   - ["Duong H. Nguyen", "https://github.com/cmpitg"]
   - ["Keyan Zhang", "https://github.com/keyanzhang"]
+  - ["Derek Schuster", "https://github.com/derrreks"]
```

Comment:
You should only add yourself to this list if you have contributed a larger section.